src/test/java/io/vertx/core/json/JacksonDatabindTest.java

```diff
@@ -32,6 +34,7 @@
  */
 public class JacksonDatabindTest extends VertxTestBase {
 
+
   @Test
   public void testGetMapper() {
```

Comment:
It seems that this file was not using the editor config because this was applied after I did auto formatting